SSA.lean

```diff
@@ -6,7 +6,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 -- ====
 import SSA.Core
 
-
```

Comment:
An unrelated change. This can be dropped.

Reply:
@tobiasgrosser I urge you to use the suggestion button :D